internal/actions/reparent.go

```diff
@@ -54,19 +54,26 @@ func Reparent(repo *git.Repo, opts ReparentOpts) (*ReparentResult, error) {
 	}
 
 	// Check that the parent branch actually exists
-	parentSha, err := repo.RevParse(&git.RevParse{Rev: opts.NewParent})
+	parentBranch := opts.NewParent
+	if opts.NewParentTrunk {
+		parentBranch = "remotes/origin/" + opts.NewParent
```

Comment:
I'm not sure that this is what we want by default. It will be surprising to people that don't use the "no local `main` branch" workflow. Requiring users to pass `origin/main` explicitly will probably be better.

Reply:
Actually I think I changed my mind on this. Assuming you always want to update on the latest version of trunk is probably reasonable. We can always add a `--no-fetch` type flag if anyone ever needs that.